projects/inc/transmitter.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+#include <array>
+#include "cobs.h"
+#include "ch.h"
+#include "simulation.pb.h"
+
+namespace message {
+class Transmitter {
+    public:
+        Transmitter();
+        void start(tprio_t priority = (NORMALPRIO + 1));
```

Comment:
I think it is better to leave out the default value for the argument. Whoever is using this class should think about the priority.